redisinsight/api/src/modules/auth/window-auth/strategies/abstract.window.auth.strategy.ts

```diff
@@ -0,0 +1,5 @@
+import { IWindowAuthStrategy, IWindowAuthStrategyData } from '../window.auth.strategy.interface';
+
+export abstract class AbstractWindowAuthStrategy implements IWindowAuthStrategy {
+  abstract isWindowExists(data: any): Promise<IWindowAuthStrategyData>;
```

Comment:
Do we plan to get data: any as an input?
also I prefer to have isAuthorized method which returns boolean

Reply:
changed